src/sentry/eventstream/kafka/backend.py

```diff
@@ -69,9 +71,17 @@ def run_post_process_forwarder(
         synchronize_commit_group,
         commit_batch_size=100,
         initial_offset_reset="latest",
+        event_type=None,
     ):
         logger.debug("Starting post-process forwarder...")
 
+        topic_config = None
+
```

Comment:
These lines seem unnecessary — this local is immediately re-assigned without referencing.